qcschema/dev/QCSchema.schema

```diff
@@ -0,0 +1,1575 @@
+            "type": "object",
+            "properties": {
+                "wavefunction": {
+                    "$ref": "#/definitions/WavefunctionProtocolEnum"
+                },
+                "stdout": {
+                    "title": "Stdout",
+                    "description": "Primary output file to keep from the computation",
+                    "default": true,
+                    "type": "boolean"
+                },
+                "error_correction": {
+                    "title": "Error Correction",
+                    "description": "Policies for error correction",
+                    "allOf": [
+                        {
+                            "$ref": "#/definitions/ErrorCorrectionProtocol"
+                        }
+                    ]
+                }
+            },
+            "additionalProperties": false
+        },
+        "AtomicInput": {
+            "title": "AtomicInput",
+            "description": "The MolSSI Quantum Chemistry Schema\n\nParameters\n----------\nid : str, Optional\n    The optional ID for the computation.\nschema_name : ConstrainedStrValue, Default: qcschema_input\n    The QCSchema specification this model conforms to. Explicitly fixed as qcschema_input.\nschema_version : int, Default: 1\n    The version number of ``schema_name`` to which this model conforms.\nmolecule : :class:`Molecule`\n    The molecule to use in the computation.\ndriver : {energy,gradient,hessian,properties}\n    Allowed computation driver values.\nmodel : :class:`Model`\n    The computational molecular sciences model to run.\nkeywords : Dict[str, Any], Default: {}\n    The program-specific keywords to be used.\nprotocols : :class:`AtomicResultProtocols`, Optional\n    Protocols regarding the manipulation of computational result data.\nextras : Dict[str, Any], Default: {}\n    Additional information to bundle with the computation. Use for schema development and scratch space.\nprovenance : :class:`Provenance`, Optional\n    Provenance information.",
+            "type": "object",
+            "properties": {
+                "id": {
+                    "title": "Id",
+                    "description": "The optional ID for the computation.",
+                    "type": "string"
+                },
+                "schema_name": {
+                    "title": "Schema Name",
+                    "description": "The QCSchema specification this model conforms to. Explicitly fixed as qcschema_input.",
+                    "default": "qcschema_input",
+                    "pattern": "^(qc_?schema_input)$",
```

Comment:
early versions did `qc_`. harmless to allow old instances to validate if they are otherwise compatible. implementations should reset to the new `qcschema_input` name. I ought to add that to the desccription.